.mk/ocp.mk

```diff
@@ -25,7 +25,10 @@ ocp-deploy: deploy-all ocp-expose-all ## OCP deploy (loki, grafana, example-cr a
 ocp-undeploy: undeploy-all  ## OCP cleanup
 
 .PHONY: ocp-run
-ocp-run: ocp-undeploy ocp-deploy   ## OCP-deploy + run the operator locally
+ocp-run: ocp-undeploy ocp-deploy ocp-deploy-flp  ## OCP-deploy + run the operator locally
+
+.PHONY: ocp-deploy-flp
+ocp-deploy-flp: ## run flp from the operator
```

Comment:
?????

Reply:
This is so we can load and unload flp without bringing up/down the entire environment whenever we make an update to the flp reconciler.

Reply:
Changed name to ocp-deploy-operator